backend/service/topology/querybuilder.go

```diff
@@ -0,0 +1,115 @@
+package topology
+
+import (
+	"fmt"
+	"strings"
+
+	sq "github.com/Masterminds/squirrel"
+
+	topologyv1 "github.com/lyft/clutch/backend/api/topology/v1"
+)
+
+const (
+	column   = "column."
+	metadata = "metadata."
+)
+
+func paginatedQueryBuilder(
+	filter *topologyv1.SearchTopologyRequest_Filter,
+	sort *topologyv1.SearchTopologyRequest_Sort,
+	pageToken,
```

Comment:
In the proto PR for this you had suggested it be an int? Which is when i changed it from a string -> int.

Reply:
suggested limit be uint, not page_token

Reply:
https://github.com/lyft/clutch/pull/779#discussion_r556047961

Reply:
sorry for the confusion. i think i may have started my review on a version with both a page_size and limit variable that was force pushed over? anyways it's not important. the page token should be a string so that you can encode arbitrary data in it if necessary in the future. per google's page it also allows backwards/forwards compatibility. "Page token contents should be a url-safe base64 encoded protocol buffer. This allows the contents to evolve without compatibility issues."

Reply:
It's fine I changed it back, no big deal.